avocado/core/test.py

```diff
@@ -313,11 +318,13 @@ def _get_tagged_name(self, logdir):
             name += ".%s" % self.tag
         tag = 0
         tagged_name = name
-        while os.path.isdir(os.path.join(logdir, tagged_name)):
+        safe_tagged_name = astring.string_to_safe_path(tagged_name)
+        while os.path.isdir(os.path.join(logdir, safe_tagged_name)):
             tag += 1
             tagged_name = "%s.%s" % (name, tag)
-        self.tag = "%s.%s" % (self.tag, tag) if self.tag else str(tag)
+            safe_tagged_name = astring.string_to_safe_path(tagged_name)
 
+        self.tag = "%s.%s" % (self.tag, tag) if self.tag else str(tag)
         return tagged_name
```

Comment:
Because then, a test that a user runs like this:

```
$ avocado run /bin/true
```

Will be shown as:

```
JOB ID     : 8d62904e29c463d52d6a412cb6c498ed00e1b736
JOB LOG    : /home/cleber/avocado/job-results/job-2016-03-11T11.12-8d62904/job.log
TESTS      : 1
 (1/1) _bin_true: PASS
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
JOB HTML   : /home/cleber/avocado/job-results/job-2016-03-11T11.12-8d62904/html/results.html
```

That was something I mentioned during the last sprint call. The feedback I got was that test names shouldn't change in the UI to reflect the need for filesystem safety in their names.